packages/components/workbench/src/WorkbenchSidebar/WorkbenchSidebar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { cx } from 'emotion';
+import { CommonProps, PropsWithHTMLElement } from '@contentful/f36-core';
+
+import { getWorkbenchSidebarStyles } from './WorkbenchSidebar.styles';
+
+export interface WorkbenchSidebarOwnProps extends CommonProps {
+  children: React.ReactNode;
+  position?: 'left' | 'right';
```

Comment:
Shouldn't we add a description and default value? Also for the other components.

Reply:
the default should be 'left', I thought I added it in the arguments of the component... no?

I didn't add description because someone told me just to migrate and deprecate it
But you are right! I will add them

Reply:
I think that just a quick description should be fine, even tho it's deprecated it's still used, so the description would help.
But we shouldn't spend too much effort on it.